server/src/schemas/trainingSchema.ts

```diff
@@ -23,6 +23,21 @@ export interface Option {
   correct: boolean;
 }
 
+export interface ModuleItemAnswerInput {
+  moduleItemID: string;
+  selectedOptionIDs: string[];
+}
+
+export interface ModuleSubmissionInput {
+  moduleID: string;
+  userID: string;
+  answers: ModuleItemAnswerInput[];
+}
+
+export interface ModuleItemAnswers {
```

Comment:
I think it would be easier to understand if we called this `ModuleItemAnswer` instead of plural _answers_. My reasoning is that we frequently have this object as part of a list, so having the singular item have a plural name muddies the waters somewhat.